home/index.html

```diff
@@ -55,22 +55,18 @@
 
       <!-- Main component for a primary marketing message or call to action -->
       <div class="jumbotron">
-        <h1>Kobalt: a modern, versatile build system</h1>
+        <h1>A modern, versatile build system</h1>
       </div>
 
       <!-- Main content -->
       <div class="col-md-9">
-        <img src="../pics/auto-completion.png" class="img-rounded"/>
-
-<h2 class="section" id="overview">Features</h2>
-<ul class="features">
-  <li>Clean, easy-to-use syntax for build files</li>
-  <li>Build file auto-completion from your favorite IDE</li>
-  <li>Intuitive plug-in architecture</li>
-  <li>Fast builds</li>
-</ul>
-</p>
 
+<h2 class="section" id="features">Features</h2>
+
+<h2 class="section" id="buildFile" indent="1"><span class="glyphicon glyphicon-check"></span> Build file auto-completion in your IDE</h2>
+    <img src="../pics/auto-completion.png" class="img-rounded"/>
+
+<h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Clean syntax for build files</h2>
```

Comment:
I took a look at these glyphs and I find them distracting and too big. Can you remove them?